benchmark/src/renderer/sharp.ts

```diff
@@ -0,0 +1,8 @@
+import type { Render } from './types';
+import sharp from 'sharp';
+
+export const render: Render = (svg) =>
+  sharp(Buffer.from(svg))
+    .png()
+    .toBuffer()
+    .then((buf) => new Uint8Array(buf));
```

Comment:
It may be necessary to add a `resize()` to ensure that the generated PNG files are of the same size.

The complexity of generating different sizes of PNGs or raw SVGs has a significant impact on performance.